modules/bazel_sonarqube/metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+    "homepage": "https://github.com/Zetten/bazel-sonarqube",
+    "maintainers": [
+        {
+            "email": "peter.vanzetten@gmail.com",
+            "github": "zetten",
```

Comment:
@zetten Do you agree with this submission?

@adincebic Could you also add yourself here? This field is not just meant for upstream maintainers, but for those who handle BCR submissions for this module.

Reply:
Done.

Reply:
> @Zetten Do you agree with this submission?

Yes,  and many thanks @adincebic!